public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/form.php

```diff
@@ -17,16 +27,52 @@ function has_feedback_form() {
 
 /**
  * Short-circuit the content, and output the feedback form (or the session select).
+ *
+ * This assumes that `wcb_session` is the only post type supporting speaker feedback. This will need to be updated
+ * if more post type support is added.
+ *
+ * @global WP_Post $post
+ *
+ * @return string
  */
 function render( $content ) {
+	global $post;
+
+	$now = date_create( time(), wp_timezone() );
```

Comment:
```suggestion
	$now = date_create( 'now', wp_timezone() );
```

This was `false` on my local env, the docs for [DateTime](https://www.php.net/manual/en/datetime.construct.php) say you should use `"now"` to get the current time with a timezone.